parent/pom.xml

```diff
@@ -1428,6 +1430,13 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-dependency-plugin</artifactId>
             <executions>
+              <execution>
+                <id>go-offline</id>
+                <phase>initialize</phase>
+                <goals>
+                  <goal>go-offline</goal>
+                </goals>
+              </execution>
```

Comment:
This execution makes sure that we actually "go-offline" when the profile is activated